go.mod

```diff
@@ -1,17 +1,41 @@
 module github.com/snyk/snyk-iac-rules
 
-go 1.16
+go 1.17
```

Comment:
Similar to the readme note, what are we using module-wise that _requires_ 1.17? Bumping the go directive in mod files should generally only be done when using go-module features that require that version.

Reply:
It's due to windows/arm64 only being supported since Go 1.17: https://goreleaser.com/deprecations/#builds-for-windowsarm64

Reply:
While it's true that we don't need to bump the module-minimum version to use a higher compiler version, I agree that it makes sense to use 1.17 everywhere if we need to use it anywhere, in the spirit of tightly controlling the compiler version.

I'll leave it up to you whether or not to bump the module version. Now that I think again, it certainly can't hurt (since we build with 1.17 in prod) and nudges developers towards using the same tools as in prod.